extensions/azure/contract-negotiation-store-cosmos/src/main/java/org/eclipse/dataspaceconnector/contract/negotiation/store/CosmosContractNegotiationStore.java

```diff
@@ -78,7 +86,7 @@ public void delete(String negotiationId) {
     @Override
     public @NotNull List<ContractNegotiation> nextForState(int state, int max) {
 
-        var partitionKey = String.valueOf(state);
+        var partitionKey = "negotiationPartition1"; //TODO: make this configurable!!!
```

Comment:
Good input. Please do this before merge the PR. Just commenting so it isn't forgotten.